### PR TITLE
[UI] Stablecoin Box Grid: migrate component to chakra UI

### PR DESCRIPTION
**Pull request by shariqanwar20**
## Description
The component has been migrated to chakra UI
The emotion styled component has been removed from the code

-  **Facing issue in centering Cash Emoji when screen size is decreased**

- Each emotion component is replaced with inline style attributes using chakra UI
- Media queries have been handled using breakpoints in chakra

### Before
Screenshot (9)
Screenshot (10)
Screenshot (8)

### After
Screenshot (12)
Screenshot (11)
Screenshot (8)

## Related Issue

Refs: #6374

### CODE REVIEW

File: src/components/StablecoinBoxGrid.tsx [hunk 1]
```diff
@@ -185,77 +65,155 @@ const GridItem: React.FC<IPropsGridItem> = ({
...
+            <Flex
+              justify="space-between"
+              marginTop={8}
+              direction={{ base: "column", md: "row" }}
+            >
               {pros && (
-                <Column>
-                  <Subtitle>
+                <Box w="100%">
+                  <Text
```
Comment (TylerAPfledderer) [inline src/components/StablecoinBoxGrid.tsx:138]: Also should be a `Heading` with an `h4` similar to previous

File: src/components/StablecoinBoxGrid.tsx [hunk 2]
```diff
@@ -185,77 +65,155 @@ const GridItem: React.FC<IPropsGridItem> = ({
...
-                  </Body>
-                </Column>
+                  </Box>
+                </Box>
               )}
               {cons && (
-                <Column>
-                  <Subtitle>
+                <Box w="100%">
+                  <Text
```
Comment (TylerAPfledderer) [inline src/components/StablecoinBoxGrid.tsx:161]: Also should be a `Heading` with an `h4` similar to previous

File: src/components/StablecoinBoxGrid.tsx [hunk 3]
```diff
@@ -185,77 +65,155 @@ const GridItem: React.FC<IPropsGridItem> = ({
...
-                  </Body>
-                </Column>
+                  </Box>
+                </Box>
               )}
-            </Row>
+            </Flex>
             <div>
-              <Subtitle>
+              <Text
```
Comment (TylerAPfledderer) [inline src/components/StablecoinBoxGrid.tsx:183]: Also should be a `Heading` with an `h4` similar to previous

File: src/components/StablecoinBoxGrid.tsx [hunk 4]
```diff
@@ -285,7 +243,16 @@ const StablecoinBoxGrid: React.FC<IProps> = ({ items }) => {
   }
 
   return (
-    <Grid>
+    <Container
+      gridTemplateColumns={"3fr 1fr"}
+      gridTemplateRows={"3fr 3fr"}
+      borderRadius="2px"
+      margin={0}
```
Comment (TylerAPfledderer) [inline src/components/StablecoinBoxGrid.tsx:250]: Should be `margin="4rem 0"` to have the vertical spacing around the container.

**Review by TylerAPfledderer — CHANGES_REQUESTED** [review]
Hey @shariqanwar20! Nice work! I had a chance to look through your PR and left some comments. 😃 

I think there are many spots where reusable child components should be recreated (Like what was originally labeled `Subtitle`, `Row`, `Column`, etc. to make sure the code is DRY. 😸

**Review by pettinarip — CHANGES_REQUESTED** [review]
Good job @shariqanwar20

**Comment by shariqanwar20** [comment]
@pettinarip @TylerAPfledderer Thanks for the feedback. I will start working on the changes right away

**Review by pettinarip — APPROVED** [review]
Thanks @shariqanwar20! I've pushed some minor fixes but overall this looks great.

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by pettinarip** [comment]
@all-contributors please add @shariqanwar20 for code

**Pull request merged**